src/stores/thumbnails.js

```diff
@@ -0,0 +1,154 @@
+import { defineStore } from 'pinia'
+import { LRUCache } from 'lru-cache'
+import { deleteCachedImageFromBasename, getImageFromBasename, getAllCacheKeys } from '@/utils/imageLoader'
+
+
+const cacheOptionsSmall = {
+  max: 5000,
+  dispose: (value, key, reason) => {
+    if (reason != 'set'){
+      deleteCachedImageFromBasename('small', key)
+    }
+  },
+  noUpdateTTL: true,
```

Comment:
This lets us re-set the values of cache keys on reload without updating their TTL, which is necessary to do because generated URLs for the cached blobs are only valid for the lifetime of your browser window, i.e. they expire on reload.